highfidelity/hifi
# Case 22363: Setting mac window size to 1/2 screen resoluion on first run.

https://highfidelity.manuscript.com/f/cases/22363/Mac-screen-size-on-first-start-needs-to-be-5-full-screen-resolution

Following other Apple apps, we are forcing the screen to open at 1/2 screen resolution on the first run. The user is allowed to change this to what he/she sees fit for their preference. On second and other consecutive runs, the user changes should stay.

Test case:
1. First open the app is at  1/2 screen size
2. Resize screen to whatever you wish
3. Close and open app
4. Second and other consecutive open should open at same size as previously set
